src/services/completions.ts

```diff
@@ -1101,16 +1107,22 @@ namespace ts.Completions {
             insertText = printer.printSnippetList(ListFormat.CommaDelimited | ListFormat.AllowTrailingComma, factory.createNodeArray([method], /*hasTrailingComma*/ true), sourceFile);
         }
 
+        const signaturePrinter = createPrinter({
+            removeComments: true,
+            module: options.module,
+            target: options.target,
+            omitTrailingSemicolon: true,
+        });
         const methodSignature = factory.createMethodSignature(
             method.modifiers,
-            method.name,
+            /*name*/ "",

```

Comment:
I would find an explanatory comment helpful here.  I believe we're dropping the name because we just want the parameter list (and return type?)?